packages/story-editor/src/app/media/utils/useDetectBlurhash.js

```diff
@@ -101,7 +102,7 @@ function useDetectBlurHash({ updateMediaElement }) {
         const blurHash = await getBlurHashFromImage(imageSrcProxied);
         await saveBlurHash(resource.id, blurHash);
       } catch (error) {
-        // Do nothing for now.
+        trackError('blurhash_generation', error?.message);
```

Comment:
💭 If an import fails in the worker the error is swallowed here. Thought it would be a good idea to add tracking here with the webpack 5 upgrade since the worker was failing silently before.